src/attestation_types/report.rs

```diff
@@ -52,6 +62,70 @@ pub struct Body {
     pub reportdata: [u16; 32],
 }
 
+impl TryFrom<&[u8]> for Body {
+    type Error = ReportError;
+
+    fn try_from(bytes: &[u8]) -> Result<Self, Self::Error> {
+        let mut cpusvn = [0u8; 16];
+        cpusvn.copy_from_slice(&bytes[0..16]);
```

Comment:
Shouldn't there be a bounds check on `bytes`, otherwise it may panic?

Reply:
@ueno Do you mean something like `impl TryFrom<&[u8; 384]> for Body` instead, where the length is specified? I've updated it to be that way.